lib/utils/compose.coffee

```diff
@@ -619,7 +619,7 @@ class BuildProgressUI
 
 	_clear: ->
 		@_tty.deleteToEnd()
-		@_maxLineWidth = @_tty.currentWindowSize().width
+		@_maxLineWidth = @_tty.currentWindowSize()?.width
```

Comment:
I don't think you need this `currentWindowSize` is always defined still, it's just the width that might not be, and that's fine.

Reply:
Oh right, that's was the whole point, to call `currentWindowSize()` and be sure there's always width/height props, even if null/undefined.